### DISCUSSION TITLE
How to do lyrics transcripting?

### DISCUSSION THREAD

**Discussion by arpitest** [openai/whisper#1188] [Q&A]
trying to use medium model on music files, for example: https://www.youtube.com/watch?v=Ld9GkEe4sOI
but it only outputs '[MUSIC]' every 10 secs... i guess it should be forced to ignore music tag and do the recognition somehow?

**Comment by EtienneAb3d** [comment]
@arpitest, you may try this (vocals extraction, silences and noise removal):
https://github.com/EtienneAb3d/WhisperHallu

I use it in this app you may experiment with:
https://github.com/EtienneAb3d/karaok-AI
